### ISSUE TITLE
Review Field Boundary GNSS Quality Information

### ISSUE THREAD

**Issue by knelson-farmbeltnorth** [ADAPT/Standard#145]
@crakerb-ship-it  and the Field Boundary GNSS Working group have prepared the attached.  We should review vs. the current state of Field Boundary (https://adaptstandard.org/docs/field-boundary/) and GNSSSource (https://adaptstandard.org/docs/gnss-source/)

[Initial Field Boundary GNSS Attributes.pdf](https://github.com/ADAPT/Standard/files/15363054/Initial.Field.Boundary.GNSS.Attributes.pdf)

**Comment by crakerb-ship-it** [comment]
The rest of the team can chime in too, but I think what you suggest with regard to ADAPT is accurate. The list the WG put together was based on what someone receiving a boundary would want to know to make an informed decision on if it was captured with a sufficient level of accuracy for their purposes. The idea being the needs of an autonomous vehicle are likely different than someone clipping a background soil type map. WGS84 makes sense to align with the rest of ADAPT, the reason we had for including was just to know if was different so it could be handled/converted properly. But in the context of a transfer format like ADAPT I agree it would make sense to specify what will be used and not leave it open to interpretation. 

Some of the finer points about GNSS source will likely be the part to discuss in greater detail. We had a lot of discussion about knowing what correction source, down to the base station ID was used during definition. The goal being to ensure when a boundary goes from one system to another its true location on the ground stays the same regardless of correction source the machine is using, assuming they are of comparable accuracy.

**Comment by knelson-farmbeltnorth** [comment]
Per discussion in the 22 May 2024 meeting, we will remove GNSSSource from the beta schema at this time with goal of coming back to it and implementing it more fully and correctly.

**Comment by Andreasox** [comment]
Just to ensure that you understand that WGS84 do have an accuracy cost, varying over the globe according to local gravity.
To my knowledge the accuracy cost is not important for operative agriculture.

Hälsningar

Andreas Oxenstierna
Dalen Hörbyvägen 53
243 94 Höör
0730-26 97 12

**Comment by knelson-farmbeltnorth** [comment]
@Andreasox Yes, we discussed the issues with using WGS84 globally.  It was also our collective experience that any accuracy issues are not substantive for ADAPT's use cases.
